elements/lisk-elements/package.json

```diff
@@ -62,15 +62,20 @@
 		"@liskhq/lisk-api-client": "3.0.0",
 		"@liskhq/lisk-constants": "1.3.0",
 		"@liskhq/lisk-cryptography": "2.4.0",
+		"@liskhq/lisk-p2p": "0.4.0",
 		"@liskhq/lisk-passphrase": "3.0.0",
+		"@liskhq/lisk-transaction-pool": "0.2.0",
 		"@liskhq/lisk-transactions": "3.0.0",
+		"@liskhq/lisk-validator": "0.3.0",
 		"@types/node": "12.12.11"
 	},
 	"devDependencies": {
 		"@types/chai": "4.1.7",
 		"@types/expect": "1.20.3",
 		"@types/jquery": "3.3.29",
 		"@types/mocha": "5.2.5",
+		"@babel/preset-env": "7.8.3",
```

Comment:
why do we need those types?

Reply:
for some reason, babel was complaining to build..